### ISSUE TITLE
Keyboard accessibility- admin login

### ISSUE THREAD

**Issue by ajanec01** [craftcms/cms#7287]
### Using keyboard only 

Hey,

I've noticed that two elements on the admin login page are not keyboard accessible. They're _show/hide_ and _forgot your password_. They're missing the `href` attribute.

![missing href on show password](https://user-images.githubusercontent.com/47136246/102555482-c8b1ee00-40be-11eb-8495-d8b532c7deec.png)

Also, the show/hide password element would provide a bit more meaning to screen reader users if it was a `button` with `aria-pressed`.

### Additional info

- Craft version: 3.5.17.1

**Comment by missmatsuko** [comment]
Thanks for making these issues, Aron!

I happened to be working on the login page this week and opened PR #7276 including a fix for this issue. 🙂

**Comment by ajanec01** [comment]
Ahh, my bad. I only went through the list of issues and didn't find those so I thought I should raise it. Will check pr's too next time.

**Comment by brandonkelly** [comment]
#7276 has been merged in for the next release!

**Comment by brandonkelly** [comment]
Craft 3.5.18 is out now with fixes for these Login page issues ✨
